styles.css

```diff
@@ -1,151 +1,108 @@
-
-
-.container
-{
-    display: flex;
-    flex-direction: column;
-   
+.container {
+  display: flex;
+  flex-direction: column;
 }
-.box1{
-  /* My Logo */
-
-
-position: static;
-width: 76px;
-height: 20px;
-left: 12px;
-top: 10px;
-
-font-family: Poppins;
-font-style: normal;
-font-weight: bold;
-font-size: 18px;
-line-height: 20px;
-/* identical to box height, or 111% */
-
-display: flex;
-align-items: center;
-text-align: center;
-
-/* Primary/B400 */
-
-color: #6070FF;
 
-
-/* Inside auto layout */
-
-flex: none;
-order: 0;
-flex-grow: 0;
-margin: 0px 0px;
+.logo {
+  display: flex;
+  text-align: center;
+  justify-content: space-between;
+  color: #6070ff;
 }
 
-
-.box2
-{
-  
-    /* Primary Text */
-    
```

Comment:
- Take your project to the next level. Kindly, remove unused commented lines and delete blank spaces, that will make your code more readable.